attributes/default.rb

```diff
@@ -153,6 +154,7 @@
   default['chef_client']['method_dir'] = '/opt/local/lib/svc/method'
   default['chef_client']['bin_dir'] = '/opt/local/bin'
   default['chef_client']['locale'] = 'en_US.UTF-8'
+  default['chef_client']['bin_path'] = '/usr/local/sbin:/usr/local/bin:/opt/local/sbin:/opt/local/bin:/usr/sbin:/usr/bin:/sbin'
```

Comment:
Let's call this new attribute, `['env_path']` to be more clear about the intent.